sql/item_mods.sql

```diff
@@ -33630,14 +33661,69 @@ INSERT INTO `item_mods` VALUES (17295,26,10); -- RACC: 10
+INSERT INTO `item_mods` VALUES (17305,950,1);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_FIRE
+
+-- Snoll Arm
+INSERT INTO `item_mods` VALUES (17306,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17306,499,2);  -- ITEM_SUBEFFECT: SUBEFFECT_ICE_DAMAGE
+INSERT INTO `item_mods` VALUES (17306,500,10); -- ITEM_ADDEFFECT_DMG: 10
+INSERT INTO `item_mods` VALUES (17306,950,2);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_ICE
+
+-- Grenade
+INSERT INTO `item_mods` VALUES (17313,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17313,499,1);  -- ITEM_SUBEFFECT: SUBEFFECT_FIRE_DAMAGE
+INSERT INTO `item_mods` VALUES (17313,500,10); -- ITEM_ADDEFFECT_DMG: 15
+INSERT INTO `item_mods` VALUES (17313,950,1);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_FIRE
+
+-- Quake Grenade
+INSERT INTO `item_mods` VALUES (17314,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17314,499,1);  -- ITEM_SUBEFFECT: SUBEFFECT_FIRE_DAMAGE
+INSERT INTO `item_mods` VALUES (17314,500,10); -- ITEM_ADDEFFECT_DMG: 20
+INSERT INTO `item_mods` VALUES (17314,950,1);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_FIRE
+
+-- Riot Grenade
+INSERT INTO `item_mods` VALUES (17315,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17315,499,5);  -- ITEM_SUBEFFECT: SUBEFFECT_LIGHTNING_DAMAGE
+INSERT INTO `item_mods` VALUES (17315,500,10); -- ITEM_ADDEFFECT_DMG: 25
+INSERT INTO `item_mods` VALUES (17315,950,5);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_THUNDER
+
+-- Bomb Arm
+INSERT INTO `item_mods` VALUES (17316,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17316,499,1);  -- ITEM_SUBEFFECT: SUBEFFECT_FIRE_DAMAGE
+INSERT INTO `item_mods` VALUES (17316,500,10); -- ITEM_ADDEFFECT_DMG: 10
+INSERT INTO `item_mods` VALUES (17316,950,1);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_FIRE
+
 -- Gold Arrow
 INSERT INTO `item_mods` VALUES (17317,26,15); -- RACC: 15
 
+-- Fire Arrow
+INSERT INTO `item_mods` VALUES (17322,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17322,499,1);  -- ITEM_SUBEFFECT: SUBEFFECT_FIRE_DAMAGE
+INSERT INTO `item_mods` VALUES (17322,500,15); -- ITEM_ADDEFFECT_DMG: 15
+INSERT INTO `item_mods` VALUES (17322,950,1);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_FIRE
+
+-- Ice Arrow
+INSERT INTO `item_mods` VALUES (17323,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17323,499,2);  -- ITEM_SUBEFFECT: SUBEFFECT_ICE_DAMAGE
+INSERT INTO `item_mods` VALUES (17323,500,15); -- ITEM_ADDEFFECT_DMG: 15
+INSERT INTO `item_mods` VALUES (17323,950,2);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_ICE
+
+-- Lightning Arrow
+INSERT INTO `item_mods` VALUES (17324,431,1);  -- ITEM_ADDEFFECT_TYPE: DAMAGE
+INSERT INTO `item_mods` VALUES (17324,499,5);  -- ITEM_SUBEFFECT: SUBEFFECT_LIGHTNING_DAMAGE
+INSERT INTO `item_mods` VALUES (17324,500,15); -- ITEM_ADDEFFECT_DMG: 15
+INSERT INTO `item_mods` VALUES (17324,950,5);  -- ITEM_ADDEFFECT_ELEMENT: ELEMENT_THUNDER
```

Comment:
You forgot to give your arrows and other ranged items a proc chance. Everything has to have one.
`ITEM_ADDEFFECT_CHANCE   = 501, // Chance of an items Additional Effect or Spikes`

Reply:
Added to arrows and the arms/grenades earlier